specs/scoring_spec.rb

```diff
@@ -42,21 +43,33 @@
 
   describe 'highest_score_from' do
     it 'returns nil if no words were passed' do
+      empty_array = []
+      Scrabble::Scoring.highest_score_from(empty_array).must_be_nil
     end
 
     it 'returns the only word in a length-1 array' do
+      one_word_array = ["apple"]
+      Scrabble::Scoring.highest_score_from(one_word_array).must_equal("apple")
     end
 
     it 'returns the highest word if there are two words' do
+      two_word_array = ["apple", "cat"] #apple = 9; cat = 5
+      Scrabble::Scoring.highest_score_from(two_word_array).must_equal("apple")
```

Comment:
Like we discussed in class, it would probably be wise to check multiple orderings of these words, that is, both `['apple', 'cat']` and `['cat', 'apple']`. You'd be surprised how often this sort of thing turns up a bug.